src/reporter.ts

```diff
@@ -54,22 +56,29 @@ export class Reporter {
   }
 
   reportFixtureStart(name: string, path: string, meta: RPItem): void {
+    this.testData = { ...this.testData, path, suiteName: name };
+    const codeRef = getCodeRef(path, name);
     const startSuiteObj: StartTestItemRQ = {
       name,
       type: TEST_ITEM_TYPES.SUITE,
       description: meta.description,
       attributes: meta.attributes,
+      codeRef,
     };
     const suiteId = this.client.startTestItem(startSuiteObj, this.launchId).tempId;
     this.suiteIds.push(suiteId);
   }
 
   reportTestStart(name: string, testMeta: RPItem): void {
+    this.testData = { ...this.testData, testName: name };
```

Comment:
What is the purpose of storing `testName` here?

Reply:
Yes, it's unnecessary. I have removed it.